### ISSUE TITLE
Environment variables in agent.cfg

### ISSUE THREAD

**Issue by rwuthric** [mtconnect/cppagent#310]
Hello

I was wondering if it would be possible to consider in future developments the possibility to allow the usage of environment variables in `agent.cfg`. This would be very helpful when developing deployment solutions in the form of Infrastructure as Code (IaC).

A possible example syntax where the IP address of the `VMC-3Axis` machine would come from the environment variable `VMC_HOST`
```
Adapters
{
    VMC-3Axis
    {
        Host = $(VMC_HOST)
        Port = 7878 
    }
}
``

**Comment by wsobel** [comment]
This is complete in the latest version. You can référence environment variables and other configuration options. 

version 2.2.0.5 has this feature.

**Comment by wsobel** [comment]
The syntax is a little different:

`Host = $VMC_HOST`

or

`Host = ${VMC_HOST}`

The demo/agent/agent.dock has this.

**Comment by wsobel** [comment]
This was addressed in PR #311

**Comment by rwuthric** [comment]
Oh my bad. Have missed that. Amazing! Thank you
